File: src/test/java/day6/WaitForItTest.java
package day6;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class WaitForItTest {

    @Test
    void firstStar() {
        WaitForIt waitForIt = new WaitForIt();
        assertEquals(288, waitForIt.firstStar());
    }

    @Test
    void secondStar() {
        WaitForIt waitForIt = new WaitForIt();
        assertEquals(71503, waitForIt.secondStar());
    }
}

File: src/test/kotlin/day06/WaitForItTest.kt
package day06

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

internal class WaitForItTest {
    @Test
    fun firstStar() {
        assertEquals(288, WaitForIt().firstStar())
    }

    @Test
    fun secondStar() {
        assertEquals(71503, WaitForIt().secondStar())
    }
}